modules/machine-health-checks-about.adoc

```diff
@@ -29,6 +29,9 @@ Consider the timeouts carefully, accounting for workloads and requirements.
 
 To stop the check, remove the resource.
 
+You might need to stop the check during the upgrade process because nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
```

Comment:
Maybe use a _For example_ here to make the transition between the paragraphs smoother? 

Also, should you s/might/should?

Reply:
I wonder if you can go with _machine health check_ in these two topics rather than `MachineHealthCheck`. Might be a bit easier to understand.

Reply:
I understand, but stating `MachineHealthCheck` clarifies that we are taking about the custom resource and that is where the user might need to take action, if any. This mechanism (of removing the resource) is changing in 4.9 - where we will have updates to `MachineHealthCheck` resource. I think I it would be consisted with the the upcoming changes too. 

So if it's ok, i would like to retain it as a resource than make it generic.

Reply:
https://github.com/openshift/openshift-docs/pull/35373#discussion_r692066777 Agree and rephrased.